src/dense/rosenbrock_interpolants.jl

```diff
@@ -282,7 +282,7 @@ end
 
 @muladd function _ode_interpolant!(out, Θ, dt, y₀, y₁, k,
     cache::Union{Rosenbrock5ConstantCache, Rosenbrock5Cache},
-    idxs, T::Type{Val{1}})
+    idxs, T::Type{Val{1}}, dv=nothing)
```

Comment:
why would dv show up here?

Reply:
I think we pass it as nothing to all interpolants